extensions/r/package.json

```diff
@@ -16,7 +13,7 @@
 		"grammars": [{
 			"language": "r",
 			"scopeName": "source.r",
-			"path": "./syntaxes/r.tmLanguage.json"
```

Comment:
Please keep using the .tmLanguage.json file extension. It makes it easier for us to distinguish the various JSON files.